pkg/sql/catalog/catalogkv/desc_getter.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 The Cockroach Authors.
```

Comment:
moved all of this out of `catalogkv.go`, only big new thing is `fromKeyValue` method